LayoutTests/resources/ui-helper.js

```diff
@@ -860,33 +869,33 @@ window.UIHelper = class UIHelper {
         });
     }
 
-    static scrollbarState(scroller, isVertical)
+    static scrollbarState(scroller, isVertical, internalFunctions)
```

Comment:
Rather than pass `internalFunctions` which is confusing, I think it would cleaner to have the caller pass the scrollingNodeID, which means we only need the `scrollingNodeID` and `isVertical` arguments.

Reply:
I don't think this is a desirable solution because then we would have to move the isUsingUISideCompositing() logic into each of the tests.

Reply:
Can we get `internalFunctions` as `scroller.document.defaultView.internals` ?

Reply:
Good idea. Fixed.